### PR TITLE
Features/bgm 630 deposit halo to x halo

### PR DESCRIPTION
**Pull request by ernestpascual**
## Description of Changes
- Changed all variables of sushibar to halochest
- Change all variables of sushi to halo
- Replaced contract references for useSushiBar() and renamed it to useHaloChest. Placed it under halo-hooks directory
- changed REACT_APP_HALOCHEST_ADDRESS to REACT_APP_HALOCHEST_ADDRESS_KOVAN

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDF-21)
[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDF-35)

### Developer Checklist:

* [x] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [x] My branch conforms with our naming convention i.e. `feature/SOOW-XXX-description`
* [x] All files have appropriate file headers and documentation
* [ ] ~~I have written new tests for your core changes, as applicable~~
* [ ] ~~I have successfully ran tests locally~~

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] Code style confirms to eslint/flow rules
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

**Comment by ernestpascual** [comment]
- Renamed xhalo to halohalo
- Made addresses dynamic

File: src/constants/index.ts [hunk 1]
```diff
@@ -72,11 +72,22 @@ export const HALO: ChainTokenMap = {
   )
 }
 
-// HALO contract addresses
+// HALO Token Addresses
+
+export const HALOAddress: ChainAddressMap = {
```
Comment (schystz) [inline src/constants/index.ts]: @ernestpascual consider using all caps to follow the convention used for constants name, I suggest `HALO_TOKEN_ADDRESS`

**Comment by schystz** [comment]
@ernestpascual my general feedback when re-engineering is I would advise against renaming the components, hooks, etc., but create a copy and modify that one instead. Leaving the original code as is might be helpful later when debugging (so we have another way to check if wether its a problem with original code or the code we added).

So for example `src/sushi-hooks/useSushiBar.ts → src/halo-hooks/useHaloChest.ts`, we can leave `useSushiBar.ts` alone and make a copy `useHaloChest.ts` instead, and simply use that in the components instead of `useSushiBar`. We can do the same for the other hooks & components that has been renamed.

File: src/constants/index.ts (hunk 1 above)
Comment (ernestpascual) [reply src/constants/index.ts]: gotcha

**Comment by ernestpascual** [comment]
Noted on this James, let me bring back the renamed components

**Comment by ernestpascual** [comment]
@schystz  updated the PR:
- brought back the renamed components and hooks
- renamed the variable you pointed out

Addtl:
- renamed other halochest to halohalo

File: src/pages/SushiBar/SushiDepositPanel.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React, { useState, useCallback } from 'react'
-import { Pair } from '@sushiswap/sdk'
+import { Currency, Pair } from '@sushiswap/sdk'
```
Comment (schystz) [inline src/pages/SushiBar/SushiDepositPanel.tsx]: @ernestpascual I guess the changes here are no longer relevant too right? Shall we revert this file as well?

File: src/pages/SushiBar/XSushiWithdrawlPanel.tsx [hunk 3]
```diff
@@ -8,9 +8,11 @@ import { Input as NumericalInput } from '../../components/NumericalInput'
 import { TYPE } from '../../theme'
 
 import { useActiveWeb3React } from '../../hooks'
+import { useTranslation } from 'react-i18next'
```
Comment (schystz) [inline src/pages/SushiBar/XSushiWithdrawlPanel.tsx]: @ernestpascual revert this file too?

File: src/pages/SushiBar/index.tsx [hunk 4]
```diff
@@ -48,7 +48,7 @@ export default function Saave() {
               <RowBetween>
                 <div>
                   <TYPE.white fontSize={14} color={theme.text2} style={{ paddingBottom: '10px' }}>
-                    {`Stake your SUSHI into xSUSHI for ~15% APY. No impermanent loss, no loss of governance rights. Continuously compounding.`}
+                    {`Stake your SUSHI into xSUSHI for ~5% APY. No impermanent loss, no loss of governance rights. Continuously compounding.`}
```
Comment (schystz) [inline src/pages/SushiBar/index.tsx]: @ernestpascual revert this file too?

Interesting though, the APY is hardcoded?? LOL

**Comment by schystz** [comment]
Thanks @ernestpascual . Just a few more minor comments but looks good to me.

**Comment by ernestpascual** [comment]
@schystz the files you mentioned are already reverted. i copied over the original sushi files since the halo files are already on their separate folders. Also, good spot at the APY idk they hardcoded it lol

File: src/pages/SushiBar/SushiDepositPanel.tsx (hunk 2 above)
Comment (ernestpascual) [reply src/pages/SushiBar/SushiDepositPanel.tsx]: https://github.com/sushiswap/sushiswap-interface/blob/5f6374a7afc4a01ef72cec1016fc5f0c5ebd0dfb/src/pages/SushiBar/SushiDepositPanel.tsx#L2

File: src/pages/SushiBar/XSushiWithdrawlPanel.tsx (hunk 3 above)
Comment (ernestpascual) [reply src/pages/SushiBar/XSushiWithdrawlPanel.tsx]: https://github.com/sushiswap/sushiswap-interface/blob/5f6374a7afc4a01ef72cec1016fc5f0c5ebd0dfb/src/pages/SushiBar/XSushiWithdrawlPanel.tsx#L11

File: src/pages/SushiBar/index.tsx (hunk 4 above)
Comment (ernestpascual) [reply src/pages/SushiBar/index.tsx]: https://github.com/sushiswap/sushiswap-interface/blob/5f6374a7afc4a01ef72cec1016fc5f0c5ebd0dfb/src/pages/SushiBar/index.tsx#L51

**Review by schystz — APPROVED** [review]

**Pull request merged**